docs/source/operations/getting-started.rst

```diff
@@ -0,0 +1,357 @@
+
+You can use kubectl_ command to monitor the pods. Tools like `k9s <https://github.com/derailed/k9s>`_ and `k8lens <https://k8slens.dev/>`_ provide graphical interfaces to monitor the pods and get their logs.
+
+Stopping
+========
+
+To stop the Substra stack, you need to stop the 3 components (backend, orchestrator and frontend) individually.
+
+* Stop the frontend: This action depends on which option you chose during the launch:
+
+  a. local server: Stop the process running the local server (usually using CONTROL + C)
+  b. Docker:
+
+     .. code-block:: bash
+
+      docker stop DOCKER_FRONTEND_CONTAINER_NAME
+
+     | with ``DOCKER_FRONTEND_CONTAINER_NAME`` the name of the frontend container you chose during the launch
+* Stop the orchestrator:
+
+  .. code-block:: bash
+
+   cd orchestrator
+   skaffold delete
+
+* Stop the backend:
+
+  .. code-block:: bash
+
+   cd substra-backend
+   skaffold delete
+
+If this command fails and you still have pods up, you can use the following command to remove the ``org-1`` and ``org-2`` namespaces entirely.
+
+.. code-block:: bash
+
+   kubectl rm ns org-1 org-2
+
+Next steps
+==========
+
+Now you are ready to go, you are ready to run either the :doc:`/auto_examples/index` or the :doc:`Substrafl (low-level library) examples </substrafl_doc/examples/index>` (low-level library).
+
+If you are interested in more deployment options or more customised set-up, you can have a look at :doc:`/operations/deploy` or at the documentation included in the repo of substra_, substra-backend_, orchestrator_ or substra-frontend_.
+
+Troubleshooting
+===============
+
+.. note::
+   Before going further in these section, you should check the following points:
+    * Check the version of Skaffold, Helm and Docker. For example, Skaffold is released very often and sometime it introduces bugs, creating unexpected errors.
+    * Check the version of the different Substra components:
+
+      * if you are using a release you can use :ref:`the compatibility table <additional/release:Compatibility table>`.
+      * if you are using the latest commit from the ``main`` git branch, check that you are up-to-date and see if there were any open issue in the repositories or any bugfixes in the latest commits.
+
+   You can also go through :doc:`the instructions one more time </operations/getting-started>`, maybe they changed since you last saw them.
+
+Troubleshooting prerequisites
+-----------------------------
+
+This section summarize errors happening when you are not meeting the hardware requirements. Please check if `you match these <#hardware>`__ first.
+
+.. note::
+   The instructions are targeted to some specific platforms (Docker for Windows in certain cases and Docker for Mac), where you can set the resources allowed to Docker in the configuration panel (information available `here for Mac <https://docs.docker.com/desktop/settings/mac/>`__ and `here for Windows <https://docs.docker.com/desktop/settings/windows/>`__).
```

Comment:
@AurelienGasser , can you review the changes I made? (also L281 &L291)

Reply:
LGTM